dev/sh/push-docs.sh

```diff
@@ -5,8 +5,11 @@
 set -e
 
 function pushDocs () {
+  BRANCH_OR_TAG="${1}"
   git submodule add -q -f -b gh-pages https://${GH_OAUTH_TOKEN}@github.com/${TRAVIS_REPO_SLUG} ghpages
 
+  # Delete files that could exist
+  rm -rf ghpages/${BRANCH_OR_TAG}/*
```

Comment:
this file will be removed in #301

Reply:
Are you sure ?
What is bery important is that files are deleted for master builds because old sami files still exist if I do not remove them with this line.
And the rsync line is missing a delete https://github.com/googleapis/gax-php/pull/301/files#diff-9cf2000c53760d837a449f874e53f792819108d3a4bf346336d0f7d082deae2cR29
But if we add a delete it will delete already built versions.
Conclusion: delete files of current build name before build, right ?